src/tests/test_config.py

```diff
@@ -34,3 +34,8 @@ def test_safeGetInt(self):
             BMConfigParser().safeGetInt('nonexistent', 'nonexistent'), 0)
         self.assertEqual(
             BMConfigParser().safeGetInt('nonexistent', 'nonexistent', 42), 42)
+
+    def test_reset(self):
+        """Reset current config"""
+        bm = BMConfigParser()
+        self.assertEqual(bm._reset(), None)
```

Comment:
I expected something like

```python
self.assertEqual(
    BMConfigParser().safeGetInt('bitmessagesettings', 'settingsversion'), 0)

```

and `self.config = BMConfigParser()` in .`setUp()` if you separate BMConfigParser logic from singleton.